docs/features/role-based-access-control/brief.md

```diff
@@ -0,0 +1,40 @@
+# Role base access control - Brief
+
+This document aims at providing an overview of Role-based Access Control (RBAC from here on) for the Marxan platform. 
+
+In broad strokes, RBAC assumes the existence of a set of roles, which are associated with different actions that they 
+can perform in the platform. Users can then be associated with roles for specific resources, which will grant them the 
+role privileges for that specific resource.
+
+The current implementation is based on the requirements described in the [User requirements document](https://docs.google.com/document/d/16x4wQYTKov0JkO1ue6iQzJ3-RXpdgQ38ygGJWW86Zo0/edit#heading=h.58xbt2es73r2).
```

Comment:
Once document has a final form, we should move it there (either pdf or key points required for understanding the feature.

Reply:
Also see `Project solution viewer`

Reply:
+1 to moving the requirements to the repo eventually (I have marked the google doc as archived in the past and removed editing access to avoid accidental new discussions once things have been settled and moved to the repo)

Reply:
I'll move the requirements we have for now, to set a base for this implementation, and because I assume there won't be many changes from here on.

If in the future they change, I'll update this part of the document.